e2e/test/scenarios/native-filters/sql-filters-source.cy.spec.js

```diff
@@ -378,8 +452,53 @@ const getStructuredTargetQuestion = questionId => {
   });
 };
 
-const getNativeTargetQuestion = questionId => {
+const getNativeTextTargetQuestion = questionId => {
+  return getTextTargetQuestion({
+    query: "SELECT * FROM PRODUCTS WHERE EAN = {{tag}}",
+    parameter: {
+      values_source_type: "card",
+      values_source_config: {
+        card_id: questionId,
+        value_field: ["field", "EAN", { "base-type": "type/Text" }],
+      },
+    },
+  });
+};
+
+const getDimensionTargetQuestion = ({ tag, parameter }) => {
   return getTargetQuestion({
+    query: "SELECT * FROM PRODUCTS WHERE {{tag}}",
+    tag: {
+      type: "dimension",
+      "widget-type": "string/=",
+      dimension: ["field", PRODUCTS.CATEGORY, null],
+      ...tag,
+    },
+    parameter: {
+      target: ["dimension", ["template-tag", "tag"]],
+      ...parameter,
+    },
+  });
+};
+
+const getStructuredDimensionTargetQuestion = questionId => {
+  return getDimensionTargetQuestion({
+    tag: {
+      dimension: ["field", PRODUCTS.CATEGORY, null],
```

Comment:
I don't see how this is different than line 474, so technically we don't need to pass `tag` here.